app/api/users/specs/users.spec.js

```diff
@@ -148,6 +156,7 @@ describe('Users', () => {
           {
             username: 'without2fa',
             email: 'another@email.com',
+            password: 'mypass',
```

Comment:
Despite this addition is nice to check the whole object, it is not covering the main reason for the problem with the password, because it was originated at the middleware side. I think we should add a kind of integration test at the route in order to verify if users.save is called with the correct params, but since instrumentRoutes, the helper we are using for tests the api calls, is not executing the middlewares as part of the call, it's fine to omit the test for now.